src/components/AddError/AddError.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from 'react';
+import cn from 'classnames';
+
+type Props = {
+  error: string,
+};
+
+export const AddError: React.FC<Props> = ({ error }) => {
+  const [closeError, setCloseError] = useState(false);
```

Comment:
It's better to use a more descriptive name for the state variable. Consider renaming 'closeError' to 'isErrorClosed'